src/components/loader-bar/loader-bar.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import LoaderBar from ".";
+
+export const DefaultStory: ComponentStory<typeof LoaderBar> = (args) => (
+  <LoaderBar mt={2} {...args} />
+);
+
```

Comment:
```suggestion
DefaultStory.storyName = "default";
```

Reply:
Hi @DlgSHi, I don't think we need to add this as we have already specified the name of the rendered story in the `stories.mdx` file:

```javascript
<Canvas>
    <Story name="default" .../>
</Canvas>
```